traffic-control-microservices-demo/README.md

```diff
@@ -80,7 +80,16 @@ And save the following bookmarks in your browser:
 ## Demo 3: ginkgo
 
 - Ensure that "front-end-v2" is setup with some latency
-- Run ginkgo
+- Install the BDD testing framework [ginkgo](https://github.com/onsi/ginkgo#set-me-up)
```

Comment:
I would put the preparation steps in a previous section "## Install ginkgo" or something, in a similar way to the previous section "## Prepare your browser for the demo", because the installation is normally done only one time even when the demo could be executed several times.